build/CMakeLists.txt

```diff
@@ -46,6 +46,7 @@ option(TRITON_ENABLE_ENSEMBLE "Include ensemble support in server" OFF)
 # Endpoints
 option(TRITON_ENABLE_HTTP "Include HTTP API in server" ON)
 option(TRITON_ENABLE_GRPC "Include GRPC API in server" ON)
+option(TRITON_ENABLE_SAGEMAKER "Include AWS SageMaker API in server" ON)
```

Comment:
Not sure if SageMaker has a GRPC API but might be useful to call this SageMaker HTTP?

Reply:
No, sagemaker endpoint will always use HTTP network protocol. SageMaker is a protocol on its own, from the user perspective it is actually different from the HTTP / GRPC protocol that we [defined](https://github.com/triton-inference-server/server/tree/master/docs/protocol), so I think we should keep it as it is so it is not confused with HTTP protocol.

Reply:
Makes sense